src/analysis/retail/paladin/holy/modules/talents/TyrsDeliverance.tsx

```diff
@@ -0,0 +1,149 @@
+import { formatDuration, formatNumber } from 'common/format';
+import SPELLS from 'common/SPELLS';
+import TALENTS from 'common/TALENTS/paladin';
+import { SpellLink } from 'interface';
+import Analyzer, { SELECTED_PLAYER, Options } from 'parser/core/Analyzer';
+import Combatant from 'parser/core/Combatant';
+import { calculateEffectiveHealing } from 'parser/core/EventCalculateLib';
+import Events, {
+  ApplyBuffEvent,
+  FightEndEvent,
+  HealEvent,
+  RemoveBuffEvent,
+} from 'parser/core/Events';
+import Combatants from 'parser/shared/modules/Combatants';
+import BoringValueText from 'parser/ui/BoringValueText';
+import Statistic from 'parser/ui/Statistic';
+import STATISTIC_CATEGORY from 'parser/ui/STATISTIC_CATEGORY';
+
+const HEALING_INC = 0.25;
+
+class TyrsDeliverance extends Analyzer {
+  static dependencies = {
+    combatants: Combatants,
+  };
+
+  protected combatants!: Combatants;
+
+  casts: number = 0;
+  duration: number = 0;
+  lastApply: number = 0;
+
+  healing: number = 0;
+  overhealing: number = 0;
+
+  incHealing: number = 0;
+  incOverhealing: number = 0;
+
+  constructor(options: Options) {
+    super(options);
+    this.active = this.selectedCombatant.hasTalent(TALENTS.TYRS_DELIVERANCE_TALENT);
+    if (!this.active) {
+      return;
+    }
+    this.addEventListener(
+      Events.applybuff.spell([TALENTS.TYRS_DELIVERANCE_TALENT]).by(SELECTED_PLAYER),
+      this.onApply,
+    );
+    this.addEventListener(
+      Events.removebuff.spell([TALENTS.TYRS_DELIVERANCE_TALENT]).by(SELECTED_PLAYER),
+      this.onRemove,
+    );
+    this.addEventListener(Events.fightend, this.onRemove);
+
+    this.addEventListener(
+      Events.heal.spell([SPELLS.TYRS_DELIVERANCE_HEALING_INCREASE]).by(SELECTED_PLAYER),
+      this.onDirectHeal,
+    );
+    this.addEventListener(
```

Comment:
Would be nice to see this per spell

Reply:
![image](https://github.com/WoWAnalyzer/WoWAnalyzer/assets/229650/295a3d69-9938-4c01-8d6e-0c678e9071b7)